checkST2110.js

```diff
@@ -701,37 +729,43 @@ const test_20_72_2 = (sdp, params) => {
 
 const greatestCommonDivisor = (a, b) => !b ? a : greatestCommonDivisor(b, a % b);
 
+const checkExactframerate = (stream = {}) => {
+  let errors = [];
+  let frMatch = stream.exactframerate.match(frameRatePattern);
+  if (!frMatch) {
+    errors.push(new Error(`Line ${stream._line}: For stream ${stream._streamNumber}, parameter 'exactframerate' does not match an acceptable pattern, as per ST 2110-20 Section 7.2.`));
```

Comment:
I think this is my only query left... since this function is not used for ST 2110-40 check as well, do we care that the message refers always to ST 2110-20? May depend on how the spec is written... not sure if it's worth passing in the specReference = 'ST 2110-20 Section 7.2' ?

Reply:
Yes, a fair point, but in fact -40 Section 7 refers directly to -20 on exactframerate thusly:

> All Senders shall signal the Format Specific Parameter exactframerate as defined in SMPTE ST 2110-20:2021 section 7.2 to indicate the frame rate related to the ANC data in the stream

...so I felt that the reference was still valid. Good enough do you think, or more context needed?

Reply:
Yep, that's good enough to keep as is I feel!